### PR TITLE
fix es operator for es 6.2.* and fix metric index with day

### PR DESCRIPTION
**Pull request by taobaorun**
### Describe the bug

fix the issue https://github.com/sofastack/sofa-lookout/issues/59

ESOperator's rollover operation does not take effect on es6

```json
{
  "error": {
    "root_cause": [
      {
        "type": "illegal_argument_exception",
        "reason": "Rollover alias [lookout-active-metrics] can point to multiple indices, found duplicated alias [[lookout-search-metrics, lookout-active-metrics]] in index template [lookout-active-metrics-t]"
      }
    ],
    "type": "illegal_argument_exception",
    "reason": "Rollover alias [lookout-active-metrics] can point to multiple indices, found duplicated alias [[lookout-search-metrics, lookout-active-metrics]] in index template [lookout-active-metrics-t]"
  },
  "status": 400
}

```

### Expected behavior
ESOperator rollover request fail.

### Actual behavior
http status 200 ,not 400.

### Steps to reproduce
- watch the server starting logs。

### Environment

- SOFALookout version: 1.6.0
- ES version : 6.3.2

### CODE REVIEW

File: server/common/src/main/java/com/alipay/sofa/lookout/server/common/es/operation/ESOperator.java [hunk 1]
```diff
@@ -50,7 +56,7 @@ public void run() {
         // add scheduler tasks
         esOperatorBuilder.getScheduler().scheduleAtFixedRate(() -> {
             doRollOver(esOperatorBuilder.alias);
-        }, 0, 10, TimeUnit.MINUTES);
+        }, 0, 10, TimeUnit.SECONDS);
```
Comment (luyiisme) [inline server/common/src/main/java/com/alipay/sofa/lookout/server/common/es/operation/ESOperator.java]: seconds？

File: server/common/src/main/java/com/alipay/sofa/lookout/server/common/es/operation/ESOperator.java [hunk 2]
```diff
@@ -157,6 +202,7 @@ void doRollOver(String alias) {
                     RequestBody.create(MediaType.parse("application/json"),
                         rollOverContent.getBytes("UTF-8"))).build();
             Response response = client.newCall(request).execute();
+            logger.info("rollOver:{}", request.toString());
```
Comment (luyiisme) [inline server/common/src/main/java/com/alipay/sofa/lookout/server/common/es/operation/ESOperator.java]: This operation is very frequent. I think it's better to change the  log level to debug.

File: server/common/src/main/java/com/alipay/sofa/lookout/server/common/es/operation/ESOperator.java (hunk 1 above)
Comment (taobaorun) [reply server/common/src/main/java/com/alipay/sofa/lookout/server/common/es/operation/ESOperator.java]: Should MINUTES

File: server/common/src/main/java/com/alipay/sofa/lookout/server/common/es/operation/ESOperator.java (hunk 2 above)
Comment (taobaorun) [reply server/common/src/main/java/com/alipay/sofa/lookout/server/common/es/operation/ESOperator.java]: Change to bug level

**Review by luyiisme — APPROVED** [review]

**Pull request merged**